frame/system/src/lib.rs

```diff
@@ -542,6 +570,11 @@ decl_module! {
 		}
 
 		/// Set some items of storage.
+		/// 
+		/// # <weight>
+		/// - `O(I)` where `I` length of `items`
+		/// - `I` storage writes (codec `O(1)`).
```

Comment:
put_raw doesn't call codec it just insert raw values. at raw keys.

The only codec is for decoding `items: Vec<KeyValue>` which actually depends on the length of keys and values but keys and values are `Vec<u8>` which is optimized in scale-codec so I think it is ok to approximate with `O(I)` anyway keys and values are expected to be usual key and values, so at the size benchmarked.

```suggestion
		/// - `I` storage writes (`O(1)`).
```